examples/visualizations/vis_pressure_field.py

```diff
@@ -11,10 +11,22 @@
 
 
 highlight_isect_idx = None
-show_broad_phase = False
+show_broad_phase = True
+
+test_Bug = True
+if test_Bug:
```

Comment:
Again, mixed camel case and snake case. My suggestion: `test_bug`.

Anyway (again), this shouldn't go to master. We might need a unit test for it though!